recipes/data-management/cygnus/ha/readme.md

```diff
@@ -80,11 +85,22 @@ By now, the data sent by the script has been processed by cygnus and will be ava
 
 Having cygnus running as a service on a Docker Swarm cluster, scaling it can be achieved as with any other docker service. For more details, refer to the [Orion recipe](../../context-broker/ha/readme.md) to see how this can be done with Docker. Otherwise, refer to the [Docker service docs](https://docs.docker.com/engine/swarm/swarm-tutorial/scale-service/).
 
-## What if I wanted a different backend?
+## Customisations
+
+### What if I wanted a different backend?
 
 If you wanted to try a different backend for your cygnus deployment, there are 3 steps you need to follow.
```

Comment:
wanted -> want

Reply:
In these cases (expressing possibility) the correct is to use the past tense :)

Reply:
Not sure: http://www.ef.com/english-resources/english-grammar/conditional/
and in any case, you haven't been consistent :)

Reply:
I think the link you provide clearly explains the difference. This is type 2, it's not a suggestion (invitation) but rather an hypothetical scenario. Maybe [this helps](https://dictionary.cambridge.org/grammar/british-grammar/conditionals-and-wishes/suppose-supposing-and-what-if) to make my point. Sorry, why do you say I haven't been consistent?